formica.go

```diff
@@ -1,3 +0,0 @@
-package main
```

Comment:
. @teemow was a loud advocate of having the main function in a file named after the tool, so actually `formicactl.go`. Is reasoning was that other tools like etcd, fleet etc do it like this too. Although nowadays fleet has a `fleetctl` and `fleetd` folder for the two binaries it creates. And in there is a file `<name>.go`.